tests/integration/cri-containerd/integration-tests.sh

```diff
@@ -625,10 +628,12 @@ function main() {
 	# TestContainerSwap
 
 	# TODO: runtime-rs doesn't support memory update currently
-	if [ "$KATA_HYPERVISOR" != "dragonball" ]; then
-		TestContainerMemoryUpdate 1
-		TestContainerMemoryUpdate 0
-	fi
+	# if [ "$KATA_HYPERVISOR" != "dragonball" ]; then
+	# 	TestContainerMemoryUpdate 1
+	# 	TestContainerMemoryUpdate 0
+	# fi
+
+	# TestContainerMemoryUpdate 0
```

Comment:
You want to remove this, no?

Reply:
Hi @fidencio Honestly, I'm not sure if this is the right way to do it, or if there's a better way to solve the CI error issue. The current situation is that `runtime-rs/qemu` does not have this feature implemented.